.github/workflows/pr-validation.yml

```diff
@@ -18,8 +18,8 @@ jobs:
         run: npm ci
       - name: Bootstrap project
         run: npm run bootstrap
-      - name: Run test for the patchset
-        run: USE_REACT_STRICT_MODE=1 npm run test:patchset
+#      - name: Run test for the patchset
```

Comment:
The full comment block can be deleted imo. Git preserves it anyways.

Reply:
I think this comment is temporary to get a preview build. should be put back

Reply:
Indeed, this was here only for testing, already removed